### ISSUE TITLE
Entity Type Select List Empty for eu Domains

### ISSUE THREAD

**Issue by kess78** [openprovider/OP-WHMCS7#57]
Hello guys,
during the registration procedure for a new .eu domain, it asks the customer to select the Entity Type to be applied to the domain.
Unfortunately the select list is empty, and no error is displayed.

If you need any further info in order to investigate, please feel free to ask.

Thank you, bye Kess.

![New_Domain_Entity_Type_Empty](https://user-images.githubusercontent.com/46758030/66377226-6be28000-e9b1-11e9-8857-d212864f41a6.png)

**Comment by soberoctober** [comment]
Hi @kess78 

Hm..that is quite strange considering the default fields should be overwritten with Openprovider's.
We are investigating this issue.

Meanwhile I recommend commenting out this field in `/whmcs/resources/domains/dist.additionalfields.php` since .eu doesn't require it.

**Comment by kess78** [comment]
Well, instead of modifying a WHMCS core file, I preferred to add the following to `resources/domains/additionalfields.php`

```
$additionaldomainfields[".eu"][] = array(
    'Name' => 'Entity Type',
    'Remove' => true,
);
```

Just after:
`$additionaldomainfields = openprovider_additional_fields();`

It seems working as expected, however now the question. Why did WHMCS add this field for .eu domains ?

**Comment by soberoctober** [comment]
resolved in [v3.1](https://github.com/openprovider/OP-WHMCS7/releases/tag/v3.1)
